parameter/FloatingPointParameterType.cpp

```diff
@@ -0,0 +1,349 @@
+/*
+ * Copyright (c) 2014-2015, Intel Corporation
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without modification,
+ * are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation and/or
+ * other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its contributors
+ * may be used to endorse or promote products derived from this software without
+ * specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, Value, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ * ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+#include "FloatingPointParameterType.h"
+#include <sstream>
+#include <iomanip>
+#include "ParameterAccessContext.h"
+#include "ConfigurationAccessContext.h"
+#include <limits>
+#include <climits>
+#include "convert.hpp"
+#include "Utility.h"
+
+using std::string;
+
+CFloatingPointParameterType::CFloatingPointParameterType(const string& strName)
+    : base(strName),
+      _fMin(-std::numeric_limits<float>::max()),
+      _fMax(std::numeric_limits<float>::max())
+{
+}
+
+string CFloatingPointParameterType::getKind() const
+{
+    return "FloatingPointParameter";
+}
+
+// Element properties
+void CFloatingPointParameterType::showProperties(string& strResult) const
+{
+    base::showProperties(strResult);
+
+    // Notation
+    strResult += "Min:";
+    strResult += CUtility::toString(_fMin);
+    strResult += "\n";
+    strResult += "Max:";
+    strResult += CUtility::toString(_fMax);
+    strResult += "\n";
+}
+
+// XML Serialization value space handling
+// Value space handling for configuration import
```

Comment:
that's not doxygen. the doxygen has been added in CParameterType's header in a previous commit. I'll remove this comment.